src/PKSim.Core/Services/CKDDiseaseStateImplementation.cs

```diff
@@ -111,6 +162,53 @@ private void updateParameter(IParameter parameter, double value)
          parameter.IsFixedValue = false;
       }
 
+      public Individual CreateBaseIndividualForPopulation(Individual originalIndividual)
+      {
+         //we need to create a new individual WITHOUT CKD and set all percentiles as in the original individuals. Other parameters, value wil be taken as is
+         var originData = originalIndividual.OriginData.Clone();
+
+         //remove the disease state to create a healthy Individual
+         originData.DiseaseState = null;
+         var healthyIndividual = _individualFactory.CreateAndOptimizeFor(originData, originalIndividual.Seed);
+
+         var allCKDParameters = parametersChangedByCKDAlgorithmAsList(healthyIndividual);
+
+         var allHealthyDistributedParameters = _containerTask.CacheAllChildrenSatisfying<IDistributedParameter>(healthyIndividual, x => !allCKDParameters.Contains(x));
+         var allOriginalDistributedParameters = _containerTask.CacheAllChildren<IDistributedParameter>(originalIndividual);
+
+         //update percentile for distributed parameters
+         foreach (var keyValue in allHealthyDistributedParameters.KeyValues)
+         {
+            var originalParameter = allOriginalDistributedParameters[keyValue.Key];
+            _parameterTask.SetParameterPercentile(keyValue.Value, originalParameter.Percentile);
+         }
+
+         //now update all other parameters that may have been changed by the user. Allowing the user to overwrite CKD Parameters is probably a recipe for disaster.
```

Comment:
I think we should differentiate and maybe indeed set some parameters to readonly in this case?
* While the TargetGFR is an input, it probably does not make much sense to overwrite parameters which are used for the calculation of the GFR: GFR_spec, Volume_Kidney, etc.
* All other parameters which are changed by CKD-Algo but not used for the GFR-Calculation (Hematocrit, etc.) - don't see a problem if the user overwites them

But this is something I would really like to discuss with Paul, Jan, etc.

Reply:
>All other parameters which are changed by CKD-Algo but not used for the GFR-Calculation (Hematocrit, etc.) - don't see a problem if the user overwites them

I think it does not make sense. I talked briefly with @aedginto about this yesterday. The publication describes changes from Healthy individual. We should stick with what we can explain. Because otherwise the algorithm will multiply a factor to the value that was changed by the user.

Reply:
To be discussed. But for me it makes sense. 
If you e.g. have a hematocrit measurement for your CKD individual - why not to allow to enter it and ovewrite the "CKD_Adjusted healthy hematocrit value"?

Reply:
For an individual yes.. But not for a population

Reply:
>Unless we say that we do not apply changes for parmeter such as HCT changed by the user and they stay as is. Which also does not makle sense because HCT is gender dependent etc..

True. This is something I would not do for sure.

>specifically because you would apply a change twice.

Maybe this is exactly the expectation? Hard to judge for me...